tony-core/src/main/java/com/linkedin/tony/TonyClient.java

```diff
@@ -393,11 +394,19 @@ public boolean init(String[] args) throws ParseException, IOException {
 
     // Set hdfsClassPath for all workers
     // Prepend hdfs:// if missing
-    hdfsClasspath = cliParser.getOptionValue("hdfs_classpath");
-    if (hdfsClasspath != null && !hdfsClasspath.startsWith(FileSystem.get(hdfsConf).getScheme())) {
-      hdfsClasspath = FileSystem.getDefaultUri(hdfsConf) + hdfsClasspath;
+    String allHdfsClasspathsString = cliParser.getOptionValue("hdfs_classpath");
+    if (allHdfsClasspathsString != null) {
+      String[] allHdfsClasspaths = allHdfsClasspathsString.split(",");
+      for (int i = 0; i < allHdfsClasspaths.length; i++) {
```

Comment:
not fully understand why this is needed. Can you make it a method and add corresponding comment so that it's more readable?